docker/.env

```diff
@@ -35,6 +35,7 @@ MF_AUTH_DB_USER=mainflux
 MF_AUTH_DB_PASS=mainflux
 MF_AUTH_DB=auth
 MF_AUTH_SECRET=secret
+MF_AUTH_LOGIN_TOKEN_DURATION="600m"
```

Comment:
maybe `MF_AUTH_LOGIN_TOKEN_EXPIRATION` is better naming

Reply:
@dusanb94 had suggested this `MF_AUTH_LOGIN_TOKEN_DURATION`

Reply:
@dusanb94 had suggested `MF_AUTH_LOGIN_TOKEN_DURATION`